### ISSUE TITLE
Resnet does not converge in training

### ISSUE THREAD

**Issue by passerbydj** [tensorflow/models#325]
Model: resnet

I am working on training the resnet-50 on imagenet dataset(not cifar), with network described in this paper (https://arxiv.org/pdf/1512.03385v1.pdf). I use the "bottleneck_residual", "batch_norm" implementation of this project, and loss does not go down in training. I find if I remove the shortcut part in "bottleneck“， everything works as normal. Other networks are working well in the setup, so the problem is probably on the resnet.py.
[resnet.zip](https://github.com/tensorflow/models/files/419922/resnet.zip)

**Comment by passerbydj** [comment]
This problem fixed as I update to version 0.10rc. But 0.10rc runs obviously slower than 0.9.

**Comment by argman** [comment]
@passerbydj , why 0.9 is slower than 0.10rc ? and how much slower ? tks.

**Comment by prb12** [comment]
Note: There are some known performance regressions in 0.10rc, which should be fixed by the release.  If the bug is fixed then hopefully all well be fine in 0.10.   Please reopen if this is not the case.
